### PR TITLE
ADD handle validation set for the first iteration

### PR DESCRIPTION
**Pull request by mathildekretz**
### What does this PR do?

*(Briefly describe the main purpose of this PR in one or two sentences. What problem does it solve or what feature does it add?)*

Add the validation dataset handle for the first iteration. Meaning the validation set is understood and copied into the proper validation data directory to be used during deepmd training.
Does not handle the validation datasets created by the other iteration of the active circle. 

---

### How to test this?

*(Provide clear, step-by-step instructions on how the reviewer can manually test your changes.)*

1. Add a dataset under the DeepMD format named *test_SYSNAME* in the *data/* directory
2. Define test data in the *user_files/dptrain_2.1.json* :
```
 "training": {... 
        "validation_data": {
            "systems": "TEST_DATA",
            "set_prefix": "set",
            "batch_size": 5,
            "auto_prob": "prob_sys_size",
            "sys_probs": null }
```
3. Run the first training, this validation set will be used during the training, you'll be able to follow the validation loss in the *lcurve.out* file written by DeepMD.

No test implemented

---

### Related Issues/Tickets

*(Link to any relevant issues.)*

--

### CODE REVIEW

**Review by EmanuelMancio — APPROVED** [review]
Looks good!
